tasks/oracle.py

```diff
@@ -0,0 +1,77 @@
+import os
+from time import sleep
+
+from invoke import task
+from invoke.exceptions import Exit
+
+
+@task
+def test(ctx, verbose=False) -> None:
+    """
+    Runs oracle functional tests against a containerized database.
+    """
+
+    if not os.environ.get("CI") and not os.environ.get("SKIP_DOCKER"):
+        start_docker(ctx, verbose)
+
+    try:
+        # ctx.run("docker ps")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion

```